libres/lib/include/ert/python.hpp

```diff
@@ -53,6 +53,7 @@ struct Submodule {
 namespace py_conversion {
 
 enkf_main_type *to_enkf_main_type(py::object obj);
+enkf_main_type *to_enkfmain_type(py::object obj);
```

Comment:
Something strange going on here. The `to_enkf_main_type` function returns a EnkfFsManager but casts to a enkf_main_type?? 

I had to add a function that actually returns a enkf_main_type but need a proper name for it. Perhaps its better to do some renaming

Reply:
Maybe @oyvindeide can shed some light on this?

Reply:
Agree that this looks a bit strange, but is a result how cwrap does the binding. Both are `enkf_main_type` as there is no `enkf_fs_manager_type`. Maybe @pinkwah has a better solution to converting the BaseCClass to the corresponding type though?

Reply:
I will leave this for now and create an issue for later.